src/bokeh/models/formatters.py

```diff
@@ -367,21 +367,21 @@ def __init__(self, *args, **kwargs) -> None:
 
     Additionally available variables are:
 
-      * ``ticks``, an array of all axis ticks as positioned by the ticker,
-      * ``index``, the position of ``tick`` within ``ticks``, and
-      * the keys of ``args`` mapping, if any.
+    * ``ticks``, an array of all axis ticks as positioned by the ticker,
+    * ``index``, the position of ``tick`` within ``ticks``, and
+    * the keys of ``args`` mapping, if any.
 
     Finding yourself needing to cache an expensive ``ticks``-dependent
     computation, you can store it on the ``this`` variable.
 
-    Example:
+    Example
```

Comment:
Is this right? I thought it was either `Example:` or `Examples:` (i.e. with the colon) c.f. 

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html